### PR TITLE
feat:added landing page

### PR DESCRIPTION
**Pull request by talrejah02**
please review and suggest changes

### CODE REVIEW

**Review by kshitijthatte — COMMENTED** [review]
This looks good, but are you using your hosted component library items?

**Review by SaurabhChawda — APPROVED** [review]
Looks good to me. but you should work on responsiveness.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+.category-card-img {
+    width: 100%;
+    height: 100%;
+    border-radius: 7px;
+}
+
+.category-card-title {
+    position: absolute;
+    bottom: 90px;
```
Comment (Jia-A) [inline style.css:127]: you can change this bottom value to rem since it is bigger than 10px.

**Review by Jia-A — APPROVED** [review]
Code looks clean. Good use of semantic HTML. Can be merged.

**Comment by talrejah02** [comment]
> This looks good, but are you using your hosted component library items?

I am using components from hosted cl but I wanted to customize it so I used it manually.
is it fine or is it compulsory to use hosted ones with links

**Pull request merged**